lib/good_bread/orders/shipment.ex

```diff
@@ -0,0 +1,19 @@
+defmodule GoodBread.Orders.Shipment do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "shipments" do
+    field :status, :string
+    field :delivery_date, :string
+    belongs_to :subscription, GoodBread.Accounts.Subscription
+    belongs_to :user, GoodBread.Accounts.User
```

Comment:
Figured adding the user would make querying the users shipments later easier, might be unnecessary though since a user has subscriptions and a subscription has shipments